config/authors/hannydevelop.json

```diff
@@ -1,6 +1,7 @@
 {
-  "name": "hannydevelop",
+  "name": "Ugochi Ukpai",
   "bio": "",
```

Comment:
I am Ugochi Ukpai, a Marine engineer, technical writer and Rust developer. I love to share knowledge about my transition from marine engineering to software development in the form of writing, to encourage people who love software development.